homeassistant/components/vicare/climate.py

```diff
@@ -136,24 +135,22 @@ class ViCareClimate(ViCareEntity, ClimateEntity):
     _attr_min_temp = VICARE_TEMP_HEATING_MIN
     _attr_max_temp = VICARE_TEMP_HEATING_MAX
     _attr_target_temperature_step = PRECISION_WHOLE
+    _attr_translation_key = "heating"
+    _attributes: dict[str, Any] = {}
```

Comment:
Mutable objects should not be initialized as a class attribute unless explicitly required. It's very error prone since all instances will modify the same object.

Reply:
reverted with https://github.com/home-assistant/core/pull/124613/commits/f98bc3bbba5054cfb50f4e39410f7a51069c1768